src/app/data/constants.js

```diff
@@ -47,11 +47,13 @@ const noticePreferenceMapping = {
   '-': 'None',
 };
 
+const CLOSED_LOCATION_REGEX = /\(CLOSED\)|STAFF ONLY|SCHWARZMAN|Performing Arts|^[^a-z]+$/;
```

Comment:
it might make sense for this to be an environment variable in case the closed locations change.